src/instruction.rs

```diff
@@ -127,7 +127,8 @@ pub enum AddressingMode {
     Absolute,         // 3    JMP $1000    full 16-bit address
     AbsoluteX,        // 3    STA $1000,X  full 16-bit address plus X register
     AbsoluteY,        // 3    STA $1000,Y  full 16-bit address plus Y register
-    Indirect,         // 3    JMP ($1000)  jump to address stored at address
+    BuggyIndirect,    // 3    JMP ($1000)  jump to address stored at address
```

Comment:
Perhaps we should add a note about what this is for here? Just to mention the hardware bug explicitly. We could add a link to the issue as well.